examples/next/faust-nx/wp-templates/index.ts

```diff
@@ -1,9 +1,11 @@
-import page from "./page";
-import archive from "./archive";
-import single from "./single";
+import page from './page';
+import archive from './archive';
+import single from './single';
+import frontPage from './front-page';
 
 export default {
   page,
   archive,
   single,
+  'front-page': frontPage,
```

Comment:
I am wondering if we should also allow the camelCase version of the template name in addition to the traditional kebab case WordPress defines for those developers who prefer shorthand? 🤔 This would benefit those that have a large amount of custom templates. _e.g. `page-*`, `single-*`, etc._

Reply:
@josephfusco I really like this idea. It would avoid the awkward behavior of defining the object template key as a string. Care to create a seperate ticket for this in the backlog?